src/components/Rows/Cells/HeaderCell.js

```diff
@@ -60,8 +62,14 @@ export default class HeaderCell extends PureComponent {
           />
         ) : (
           <Fragment>
-            {renderElement(renderer, this.getRequiredProps(), title)}
-
+            <span onClick={this.handleSort}>
+              {renderElement(renderer, this.getRequiredProps(), title)}
+            </span>
+            <Filter
```

Comment:
We should conditionally render this, if and only if `filterRenderer` is present.